api/app/helpers/spree/api/api_helpers.rb

```diff
@@ -121,12 +121,18 @@ def required_fields_for(model)
         :id, :number, :state, :order_id, :memo, :created_at, :updated_at
       ]
 
-      @@address_attributes = [
-        :id, :firstname, :lastname, :full_name, :address1, :address2, :city,
-        :zipcode, :phone, :company, :alternative_phone, :country_id, :country_iso,
-        :state_id, :state_name, :state_text
+      @@non_deprecated_address_attributes = [
```

Comment:
ApiHelpers explicitly lists all attributes to show in jbuilder templates, I needed the current list and the list including the deprecated ones

Reply:
I mean that we can get the list of the deprecated attributes by removing the deprecated ones from the current list. Just because we'll have less things to change if we add other attributes to address in the future.

Reply:
Sorry I misunderstood, yes that would be better 👍